src/pages/CreateFeedback.tsx

```diff
@@ -58,8 +60,19 @@ export const CreateFeedback = (props: IProps) => {
       comments: [],
     };
 
-    productRequests.push(newFeedback);
-    navigate('/');
+    if (title.length === 0 && feedback.length === 0) {
+      setTitleError('error');
+      setFeedbackError('error');
+    } else if (title.length === 0) {
+      setTitleError('error');
+      setFeedbackError('');
+    } else if (feedback.length === 0) {
+      setTitleError('');
+      setFeedbackError('error');
+    } else {
+      productRequests.push(newFeedback);
+      navigate('/');
+    }
     event.preventDefault();
```

Comment:
Typically we `preventDefault()` right at the top of the function as the first thing we do. That way we cancel any native behavior before any of our custom logic runs.

Reply:
Ah makes sense. Updated